include/odp/api/spec/classification.h

```diff
@@ -346,6 +395,12 @@ typedef struct odp_cls_cos_param {
 	 */
 	odp_cos_action_t action;
 
+	/** Enable statistics. If true, counters are incremented when packets
+	 * are classified to the CoS. Default is false. @see
+	 * odp_cls_cos_stats().
+	 */
+	odp_bool_t stats_enable;
```

Comment:
If implementation has limited number of CoS that can have stats enabled, how to indicate that implementation is out-of-stats resources? Maybe new odp_cls_cos_create() return value.

Reply:
Capability added in v6.